packages/expo-cli/src/commands/build/AndroidBuilder.ts

```diff
@@ -118,58 +118,50 @@ See https://docs.expo.io/distribution/building-standalone-apps/#2-configure-appj
       (this.options.clearCredentials || !credentialsExist)
     ) {
       console.log('');
-      const questions: Question[] = [
+      const questions: PromptObject[] = [
         {
-          type: 'rawlist',
+          type: 'select',
           name: 'uploadKeystore',
           message: `Would you like to upload a keystore or have us generate one for you?\nIf you don't know what this means, let us handle it! :)\n`,
           choices: [
-            { name: 'Let Expo handle the process!', value: false },
-            { name: 'I want to upload my own keystore!', value: true },
+            { title: 'Let Expo handle the process!', value: false },
+            { title: 'I want to upload my own keystore!', value: true },
           ],
         },
         {
-          type: 'input',
+          type: (prev, values) => values.uploadKeystore && 'text',
```

Comment:
This conditional type is the replacement of the `when` method. ([see this example](https://github.com/terkelg/prompts/blob/master/example.js#L33))

Reply:
Maybe I should use `answers`, to clarify this a bit more?

Reply:
```suggestion
          type: (prev, answers) => {
            if (answers.uploadKeystore) {
              return 'text';
            } else {
              return false;
            }
          }
```
Maybe?

Reply:
Yes, that's also a good one. But I'm afraid this might cause a lot of additional code. Going from 1 line, to mark something as optional, to 7 might be a bit much if we have to do this for everything. We can also go for an inline approach using the same explicit return values?

```
type: (prev, answers) => (values.uploadKeystore ? 'text' : false)
```